packages/node/src/subcommands/mmrMigrate.module.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020-2022 OnFinality Limited authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { Module } from '@nestjs/common';
+import { DbModule, NodeConfig, StoreService } from '@subql/node-core';
+import { Sequelize } from 'sequelize';
+import { ConfigureModule } from '../configure/configure.module';
+import { MMRMigrateService } from './mmrMigrate.service';
+
+@Module({
+  providers: [
+    {
+      provide: MMRMigrateService,
+      useFactory: (config: NodeConfig, sequelize: Sequelize) => {
+        return new MMRMigrateService(sequelize, config);
+      },
+      inject: [NodeConfig, Sequelize],
+    },
```

Comment:
Use factory is only needed if we have extra logic

```suggestion
    MMRMigrateService,
```

Reply:
> Use factory is only needed if we have extra logic

without useFactory , NodeConfig comes out undefined

Reply:
```suggestion
    NodeConfig,
    MMRMigrateService,
```